gemfirexd/shared/src/main/java/io/snappydata/thrift/common/SnappyTSocket.java

```diff
@@ -185,21 +184,22 @@ public SnappyTSocket(InetAddress srvAddress, int port, String clientId,
   /**
    * Initializes the socket object
    */
-  private static SocketChannel initSocket(boolean blocking)
-      throws TTransportException, IOException {
+  private static SocketChannel initSocket(boolean blocking) throws IOException {
     SocketChannel socketChannel = SocketChannel.open();
     socketChannel.configureBlocking(blocking);
     return socketChannel;
   }
 
+  @SuppressWarnings("SameParameterValue")
```

Comment:
nit: should we remove the `key` parameter instead of suppressing warning

Reply:
skipping for now. Will make changes separately